config/windows-containerd-monitor-filelog.json

```diff
@@ -18,8 +18,23 @@
 		},
 		{
 			"type": "temporary",
-			"reason": "IncompatibleContainer",
-			"pattern": ".*CreateComputeSystem.*"
+			"reason": "ContainerProcessStartFailed",
+			"pattern": ".*failed to start.*hcsshim::System::CreateProcess.*"
+		},
+		{
+			"type": "temporary",
+			"reason": "ContainerDContainerCreationFailed",
+			"pattern": ".*failed to create containerd container.*error unpacking image.*"
+		},
+		{
+			"type": "temporary",
+			"reason": "CorruptContainerImageLayer",
+			"pattern": ".*failed to pull and unpack image.*failed to extract layer.*"
+		},
+		{
```

Comment:
Considering there are various specific errors that contain this pattern like shown in the description, it would be good to have a more generic pattern to catch what we might miss in the specific type of errors and have the error messages sent to the exporter.

Reply:
From the examples I think `wrong diff id calculated on extraction` is the best one. `archive/tar: invalid tar header: unknown` is also a good candidate. These imply that the image was downloaded but a checksum failed or bad data was pulled.

Anything with unauthorized shouldn't raise a signal. That's generally user error (permissions) not an issue due to the node becoming damaged.

`The system cannot find the file specified.` implies that the container image itself is broken but I'm curious on how you got this to repro.

`400 Bad Request - Server message: unknown: 'digest' parameter is not a valid digest` almost feels like a node level problem but I'm thinking this can happen if a user creates an image and then deletes it and pushes a Deployment using the deleted image.

Reply:
Discussed with @jeremyje offline, and will remove this for now to avoid false positives where this error may also result from a user error. We do not necessarily want NPD to report this as an Unhealthy Node if that is the case, but only want to report what will need node remediation.